### PR TITLE
Add outreach proposal for hubs & coworking spaces

### PR DESCRIPTION
**Pull request by yawboakye**
This proposal borrows a lot from the university's with a few
modifications such as communicating exactly what we offer and what we
require of the institution.

### CODE REVIEW

File: outreach/hubs.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+My name is [NAME], and I'm writing you on behalf of the RustBridge program. RustBridge is an inclusive and accessible event designed to teach newcomers to systems programming and programming in general about the programming language Rust. We, the [LOCAL ORGANIZING GROUP], are interested in reaching more learners, and would like to collaborate with you to host a free event for members of your institution and the general public.
+
+RustBridge follows BridgeFoundry format, which is used by RailsBridge and RubyBridge as well and has proven to be especially effective around learners who come from a diverse and non-traditional background. We have organized RustBridge events at universities and university-related events before, like the HackIllinois, University of Waterloo, the Canadian University Software Engineering Conference (CUSEC), and we think that it would be a great fit for [HUBNAME] as well.
```
Comment (ghost) [inline outreach/hubs.md]: `follows the`

File: outreach/hubs.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We are proposing would be an all-day event, meaning it would span about 6-8 hours.
```
Comment (ghost) [inline outreach/hubs.md]: `What we are proposing would be [...]`

File: outreach/hubs.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+We make an extra effort to relate programming concepts in Rust to similar concepts in other languages, in order to make them easier to understand.
+
+Broadly, the event can be split into two parts: An interactive presentation about the core language concepts of Rust, using http://exercism.io as a platform. This way, learners can learn at their own pace using a well-known curriculum. The second part leads learners to build their own small website in Rust. This way, every learner can work at their own pace if they want, but still collaborate efficiently.
+
+### What We Offer (Incomplete)
+- Mentors/teachers for the workshop
+- Workshop materials
+- Food?
+- Spare laptops?
+- What's missing?
```
Comment (ghost) [inline outreach/hubs.md]: an exciting and very fun time 😉

File: outreach/hubs.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+### What We Require (Incomplete)
+- Venue (for how many learners?)
+- Main and standby power
+- Internet (minimum speed requirement?)
+- Spare laptops?
+- What's missing?
```
Comment (ghost) [inline outreach/hubs.md]: food and drinks, if possible!

File: outreach/hubs.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- Workshop materials
+- Food?
+- Spare laptops?
+- What's missing?
+
+### What We Require (Incomplete)
+- Venue (for how many learners?)
+- Main and standby power
+- Internet (minimum speed requirement?)
+- Spare laptops?
```
Comment (ghost) [inline outreach/hubs.md]: not really, having a laptop is a hard requirement as a learner and shouldn't be offered from the venue

File: outreach/hubs.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+3. to provide them with their own personal and extendable project that they can work on after they finish.
+
+We make an extra effort to relate programming concepts in Rust to similar concepts in other languages, in order to make them easier to understand.
+
+Broadly, the event can be split into two parts: An interactive presentation about the core language concepts of Rust, using http://exercism.io as a platform. This way, learners can learn at their own pace using a well-known curriculum. The second part leads learners to build their own small website in Rust. This way, every learner can work at their own pace if they want, but still collaborate efficiently.
+
+### What We Offer (Incomplete)
+- Mentors/teachers for the workshop
+- Workshop materials
+- Food?
```
Comment (ghost) [inline outreach/hubs.md]: see below, we have to rely on sponsoring to get food

**Review by ghost — COMMENTED** [review]
this is really good! just a couple of things

**Comment by yawboakye** [comment]
@oe Updated with the requested changes

**Review by ghost — APPROVED** [review]
LGTM!

**Pull request merged**